### PR TITLE
Added changes to callback controller for testing.

### PR DESCRIPTION
**Pull request by yashgit891**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: Controller/Payment/Callback.php [hunk 1]
```diff
@@ -221,29 +230,33 @@ public function execute()
             }
             catch(\Razorpay\Api\Errors\Error $e)
             {
-
+                // echo $e->getMessage();
```
Comment (garimaprachi) [inline Controller/Payment/Callback.php]: Remove echo statement if not neccessary

File: Controller/Payment/Callback.php [hunk 2]
```diff
@@ -221,29 +230,33 @@ public function execute()
...
                     ->critical("Validate: Razorpay Error message:" . $e->getMessage());
+                // @codeCoverageIgnoreEnd
                 $responseContent['message'] = $e->getMessage();
 
                 $code = $e->getCode();
             }
             catch(\Exception $e)
             {
-
+                // echo $e->getMessage();
```
Comment (garimaprachi) [inline Controller/Payment/Callback.php]: Remove echo statement if not neccessary

File: Controller/Payment/Callback.php [hunk 3]
```diff
@@ -221,29 +230,33 @@ public function execute()
...
                 $responseContent['message'] = $e->getMessage();
 
                 $code = $e->getCode();
             }
         }
         else
         {
-            $objectManager = \Magento\Framework\App\ObjectManager::getInstance();
-            $quote = $objectManager->get('Magento\Quote\Model\Quote')
+            //$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
```
Comment (garimaprachi) [inline Controller/Payment/Callback.php]: Remove commented line if not neccessary

File: Controller/Payment/Callback.php (hunk 1 above)
Comment (enishant) [reply Controller/Payment/Callback.php]: @garimaprachi Removed echo statement

File: Controller/Payment/Callback.php (hunk 2 above)
Comment (enishant) [reply Controller/Payment/Callback.php]: @garimaprachi Removed echo statement

File: Controller/Payment/Callback.php (hunk 3 above)
Comment (enishant) [reply Controller/Payment/Callback.php]: @garimaprachi Removed commented line

**Review by garimaprachi — COMMENTED** [review]
LGTM

**Review by garimaprachi — COMMENTED** [review]
LGTM

File: Controller/Payment/Callback.php [hunk 4]
```diff
@@ -60,7 +60,10 @@ public function __construct(\Magento\Framework\App\Action\Context $context, \Mag
         $this->catalogSession   = $catalogSession;
         $this->order            = $order;
 
-    }
+        $this->captureCommand = new CaptureCommand();
+        $this->authorizeCommand = new AuthorizeCommand();
+
```
Comment (abdulwahidsharief) [inline Controller/Payment/Callback.php]: @yashgit891 remove empty line

**Review by abdulwahidsharief — APPROVED** [review]
LGTM

**Pull request merged**